{
  "submission_id": "S16",
  "user_id": "U1",
  "problem_id": "P1",
  "timestamp": "2025-09-01T20:27:10Z",
  "status": "runtime_error",
  "language": "python",
  "test_results": [
    {
      "test_case_number": 1,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.0,
      "memory_usage": 7.71484375,
      "actual_output": "",
      "expected_output": "2"
    },
    {
      "test_case_number": 2,
      "status": "runtime_error",
      "message": "Runtime Error: Runtime Error (Exit Code: 1)",
      "execution_time": 0.0,
      "memory_usage": 7.671875,
      "actual_output": "",
      "expected_output": "4"
    }
  ]
}